ff/serving/metadata/metadata.go

```diff
@@ -775,6 +793,47 @@ func NewMetadataServer(logger *zap.SugaredLogger) (*MetadataServer, error) {
 	}, nil
 }
 
+func NewMetadataTypesenseServer(logger *zap.SugaredLogger, port string, host string, apiKey string) (*MetadataServer, error) {
+	logger.Debug("Creating new metadata server")
+	client := typesense.NewClient(
+		typesense.WithServer(fmt.Sprintf("http://%s:%s", host, port)),
+		typesense.WithAPIKey(apiKey))
+	logger.Debug("Creating typsense client on " + fmt.Sprintf("http://%s:%s", host, port) + "and apikey:" + apiKey)
```

Comment:
sugarredlogger already uses sprintf, just replace this line with

logger.Debug("Creating typsense client on http://%s:%s with apiKey %s", host, port, apiKey)